crates/axvm/src/arch/aarch64/mod.rs

```diff
@@ -28,14 +28,11 @@ pub fn has_hardware_support() -> bool {
 
 static INIT: Once = Once::new();
 
-pub fn register_lower_aarch64_synchronous_handler() -> AxResult {
-    INIT.call_once(|| {
-        if !register_exception_handler_aarch64(HVC_EXCEPTION, hvc_handler) {
-            panic!("Failed to register HVC handler");
-        };
-        if !register_exception_handler_aarch64(DATA_ABORT_EXCEPTION, data_abort_handler) {
-            panic!("Failed to register data abort handler");
-        }
-    });
+pub fn register_lower_aarch64_synchronous_handler_arch() -> AxResult {
```

Comment:
rename this function,

at least, it's strange to see a function named as `xxx_arch()`

Reply:
Lgtm